lib/plugins/bidirectionalData.js

```diff
@@ -181,16 +181,16 @@ function getReverseTransformations(list, values, callback) {
 }
 
 /**
- * Apply a list of transformations to the reported values, 
+ * Apply a list of transformations to the reported values,
  * generating a new array of values with the additional data.
  * For NGSI-v2 this consists of value transformations only
  *
  * @param {Array} values                List of reported attributes (with their name, type and value).
  * @param {Array} transformations       List of transformations to apply (with their name, type and expression).
  */
-function processTransformationsNGSIv2(values, transformations, callback) {
+function processTransformationsNGSIv2(values, transformations, typeInformation, callback) {
```

Comment:
`typeInformation` should be added as new `@param` in the function "JavaDoc"

Reply:
Fixed e0220ae